README.md

```diff
@@ -79,31 +184,31 @@ if enabled {
 }
 ```
 
-2. Get the value of a flag for a given feature. `TweakManager` will return the value from the configuration with the highest priority and automatically fallback to the others if no set value is found.
+2. Get the value of a flag for a given feature. `TweakManager` will return the value from the tweak provider with the highest priority and automatically fallback to the others if no set value is found.
 
 Use either `tweakWith(feature:variable:)` or the provided property wrappers.
 
 ```swift
 // check for a tweak value
 let tweak = tweakManager.tweakWith(feature: "some_feature", variable: "some_flag")
 if let tweak = tweak {
-    // tweak was found in some configuration, use tweak.value
+    // tweak was found in some tweak provider, use tweak.value
 } else {
-    // tweak was not found in any configuration
+    // tweak was not found in any tweak provider
 }
 ```
 
 `@TweakProperty` and `@OptionalTweakProperty` property wrappers are available to mark properties representing feature flags. Mind that by using these property wrappers, a static instance of `TweakManager` is needed. 
 
-```
+```swift
 @TweakProperty(fallbackValue: <#fallback_value#>,
                feature: <#feature_key#>,
                variable: <#variable_key#>,
                tweakManager: <#TweakManager#>)
```

Comment:
This needs to be updated to `@FallbackTweakProperty` or have the `fallbackValue` param removed from the signature

Reply:
Well spotted! Thanks.